docs/source/guides/libraries.rst

```diff
@@ -301,12 +272,313 @@ Python Library
 Where to Get
```

Comment:
I suggest rewriting this section using the same style as for the part about Android library.